checker/tests/src/tests/NullnessRawTypesNoCachingTest.java

```diff
@@ -0,0 +1,27 @@
+package tests;
+
+import org.checkerframework.framework.test.CheckerFrameworkTest;
+
+import java.io.File;
+
+import org.junit.runners.Parameterized.Parameters;
+
+/**
+ * Tests for the Nullness Checker on raw types with caching turned off.
+ */
+public class NullnessRawTypesNoCachingTest extends CheckerFrameworkTest {
+
+    public NullnessRawTypesNoCachingTest(File testFile) {
+        super(testFile,
+                org.checkerframework.checker.nullness.NullnessChecker.class,
+                "nullness",
+                "-Anomsgtext",
+              "-AatfDoNotCache" ,"-AatfDoNotReadCache");
+    }
+
+    @Parameters
+    public static String[] getTestDirs() {
+        return new String[]{"nullness-rawtypes-nocaching"};
```

Comment:
Why do we have two different test cases for caching and no caching? Isn't our goal to make the behavior the same between the two? I.e. have one directory of tests that get's used by the checker with and without caching.